validator/rpc/server.go

```diff
@@ -78,7 +87,8 @@ type Server struct {
 	ctx                       context.Context
 	walletDir                 string
 	jwtSecret                 []byte
-	grpcHeaders               []string
+	// Deprecated: gRPC API will still be supported for some time, most likely until v8 in 2026, but will be eventually removed in favor of REST API.
```

Comment:
do we need this? the user doesn't get affected I think, you can mention it at the top of the server structs which ones are grpc related might be overkill to slap it inside the structs like this